File: 0088-merge-sorted-array/0088-merge-sorted-array.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        vector<int> merged (m+n);
        int index = 0;
        for (int i=0; i < m; i++){
            merged[index] = nums1[i];
            index +=1;
        }
        for (int j=0; j < n; j++){
            merged[index] = nums2[j];
            index +=1;
        }
        sort(merged.begin(), merged.end());
        nums1 = merged;
    }
};

File: 0088-merge-sorted-array/0088-merge-sorted-array.py
class Solution(object):
    def merge(self, nums1, m, nums2, n):
        """
        :type nums1: List[int]
        :type m: int
        :type nums2: List[int]
        :type n: int
        :rtype: None Do not return anything, modify nums1 in-place instead.
        """
        if n != 0:  #there are elements to merge from nums2
            i = 0   #choose the 1st element of nums2, m is already pointing to the place where new numnbers should be added
            while i < n:
                nums1[m+i] = nums2[i]
                i += 1
            nums1.sort()